src/components/CommunityApps.jsx

```diff
@@ -0,0 +1,172 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { browserHistory } from "react-router";
+
+import Button from '@material-ui/core/Button';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from "@material-ui/core/styles";
+
+import Version from "./children/Version";
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        '& > *': {
+            margin: theme.spacing(3)
+        }
+    },
+	grid_container: {
+		marginTop: theme.spacing(3),
+		marginBottom: theme.spacing(10),
+		flexGrow: 1,
+	},
+    h4: {
+		// fontWeight: "bold",
+		color: theme.palette.primary,
+        marginBottom: theme.spacing(3),
+		marginTop: theme.spacing(4),
+	},
+    cardroot: {
+        maxWidth: 400,
+        marginBottom: theme.spacing(3),
+    },
+    cardmedia: {
+        height: 200,
+    },
+    footerContainer: {
+		marginTop: theme.spacing(4),
+		marginBottom: theme.spacing(4),
+		display: "flex",
+	},
+    icons: {
+		display: "flex",
+	},
+	icon: {
+		height: 80,
+		borderRadius: 5,
+		display: "flex",
+		justifyContent: "center",
+		alignItems: "center",
+		marginRight: theme.spacing(1)
+	},
+    footer: {
+		overflow: "hidden",
+		backgroundColor: "#ffffff"
+	},
+}))
+
+const CommunityApps = ({ location }) => {
+    const classes = useStyles();
+    const images = [
+        {app: "Galveston", img: "/public/galveston.jpeg"},
+        {app: "Salt Lake City", img: "/public/salt-lake-city.jpeg"},
+        {app: "Joplin", img: "/public/joplin-main-street.jpeg"},
```

Comment:
add urls
```
const images = [
        {app: "Galveston", img: "/public/galveston.jpeg", url: "/playbook/galveston"},
        {app: "Salt Lake City", img: "/public/salt-lake-city.jpeg", url: "/plabyook/slc"},
        {app: "Joplin", img: "/public/joplin-main-street.jpeg", url: "/playbook/joplin"},
    ]
```

Reply:
I added the urls but they won't work on local machine as there is nothing running. If we run a playbook server, then I think it will work.